code/datums/wires/airlock.dm

```diff
@@ -96,9 +96,9 @@
 				return
 			if(!A.requiresID() || A.check_access(null))
 				if(A.density)
-					INVOKE_ASYNC(A, /obj/machinery/door/airlock.proc/open)
+					INVOKE_ASYNC(A, /obj/machinery/door/airlock.proc/open, 1)
 				else
-					INVOKE_ASYNC(A, /obj/machinery/door/airlock.proc/close)
+					INVOKE_ASYNC(A, /obj/machinery/door/airlock.proc/close, 1)
```

Comment:
```suggestion
					INVOKE_ASYNC(A, /obj/machinery/door/airlock.proc/close, TRUE)
```

Reply:
`forced` has 3 possible values: 0 to open normally, 1 to open normally even if the open wire is cut, and 2 if you're prying it open.

Reply:
Those should be defines then, are they not already?

Reply:
They aren't, but it might be out of scope to turn them into defines in this PR.

Reply:
Yeah, that's fine then.